config/config.go

```diff
@@ -13,7 +13,8 @@ type Config struct {
 
 type BaseConfig struct {
 	LogLevel       string `mapstructure:"log-level"`
-	OracleMnemonic string `mapstructure:"doracle-mnemonic"`
+	OracleMnemonic string `mapstructure:"oracle-mnemonic"`
+	ListenAddr     string `mapstructure:"listen_addr"`
```

Comment:
It seems doracle is not accepting api requests, Where do I use this setting?

Reply:
When the listening address is not given to HTTP server, the server has default listening addr, ":http" (port 80). ([ref](https://pkg.go.dev/net/http#Server))
So I add listening addr to config, and it can be used for future admin purpose (https://github.com/medibloc/panacea-doracle/pull/3#discussion_r929439137).

Reply:
OK. I got it.